### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 6 - Esercizio 14

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#576] [Esercizi M-Z]
![es14](https://github.com/user-attachments/assets/45c4bcd2-d391-4698-ab6c-31d00e3f91fc)

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/76fb77cc-ebc2-4db4-a8c0-ac9622551b71)
